client/support/support-user/login-dialog.jsx

```diff
@@ -31,6 +31,26 @@ const SupportUserLoginDialog = React.createClass( {
 		this.setState( { supportPassword: '' } );
 	},
 
+	onInputKeyDown( event ) {
+		switch ( event.key ) {
+			case 'Enter':
+				event.preventDefault();
+				switch ( event.target.name ) {
+					case 'supportUser':
+						console.log( this.supportPasswordInput );
+						this.supportPasswordInput.focus();
+						break;
+					case 'supportPassword':
+						this.onChangeUser();
+						break;
```

Comment:
The `onChangeUser()` is a method on this class which proxies the `onChangeUser` prop, since the prop doesn't have access to the form state - see [line 30](https://github.com/Automattic/wp-calypso/pull/3243/files#diff-f4aee17c1fa41890b73a04baaa790409R30). It's definitely a bit confusing, I'll see what I can do to simplify this.